compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "2.4"
+services:
+  reverse-proxy:
+    image: bitnami/nginx:1.22.0-debian-11-r8
+    user: ${REVERSE_PROXY_CONTAINER_USER}
+    ports:
+      - "80:8080"
+      - "443:8443"
+    volumes:
+      - ${NGINX_CONF}:/opt/bitnami/nginx/conf/server_blocks/vhost.conf:ro
+  web:
+    image: bitnami/node:16.15.1-debian-11-r5
```

Comment:
This is a placeholder until the contents of https://github.com/PhilanthropyDataCommons/service/pull/14 are working. So it will be more like `image: ghcr.io/philanthropydatacommons:latest` in the near term and `image: ghcr.io/philanthropydatacommons:[version]` eventually.